ivy/functional/ivy/device.py

```diff
@@ -664,6 +664,24 @@ def set_default_device(device: Union[ivy.Device, ivy.NativeDevice]):
     device
         The device to set as the default device
 
+    Examples
+    --------
+    >>> ivy.set_default_device("cpu")
+    >>> ivy.default_device()
+    'cpu'
+    
+    >>> import torch
```

Comment:
Please move this line to above the `ivy.set_backend("torch")` line as we don't need to `import torch` in this example but need it in the next example.